bot/plugins/languages.py

```diff
@@ -0,0 +1,19 @@
+import crescent
+import hikari
+
+from bot.app import Plugin
+from bot.constants import EMBED_COLOR
+
+plugin = Plugin()
+
+
+@plugin.include
+@crescent.command
+async def languages(ctx: crescent.Context) -> None:
+    lang_names = list(
+        f"`{runtime}`" for runtime in plugin.model.manager.runtimes.run if runtime
+    )
```

Comment:
```suggestion
    lang_names = [
        f"`{runtime}`" for runtime in plugin.model.manager.runtimes.run if runtime
    ]
    lang_names.sort()
```

Reply:
I don't think sorting is a good idea. The list should already be sorted since the `.sort()` method for the manager should've been called earlier.
Maybe I should add a comment explaining this.